tests/drain_test.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+from drain3.drain import Drain
+
+
+class DrainTest(unittest.TestCase):
+    def test_add_log_message(self):
+        model = Drain()
```

Comment:
without explicit parameters you can't really guarantee stable results

Reply:
I agree with @barakreif that ctor arguments shall be specified. explicitely.

Reply:
I consider default parameter part of the contract of the class. By using default constructor in tests I can rely on them to detect regressions which violate this contract.

For example. In one of my future contributions I plan to add a new parameter which will cap the max memory consumed by this model. If I specified all the parameters in tests I could miss the fact that default behavior changes and learn about it only when this change makes to production. 

I'm happy to add a second unit tests with all the parameters set.

Reply:
OK, so please add a second file

Reply:
done :)